### ISSUE TITLE
Unwanted <p> tags in render

### ISSUE THREAD

**Issue by hynese** [apidoc/apidoc#270]
Greetings,

So I've just started using apidocjs.

I have this annotation:

```
/**
  * @api {GET} /user/:id request specific User
  * @apiSampleRequest /user
  *
  * @apiName getUser
  * @apiGroup User
  *
  * @apiExample {curl} Example usage:
  *     curl -i https://localhost/user/4711
  *
  * @apiParam {Number} id User unique ID
  *
  * @apiSuccess {String} json User object
  *
  */
```

It's rendering with funny `<p>` tags as shown:

![screen shot 2015-06-01 at 17 57 25](https://cloud.githubusercontent.com/assets/315697/7919015/58a94d2a-088d-11e5-97fe-01a4372ccced.png)

**Comment by m13** [comment]
+1

**Comment by Antiseptikk** [comment]
+1. I have the same problem

**Comment by jhaaaa** [comment]
+1

**Comment by harsain** [comment]
+1
same issue

**Comment by wisoot** [comment]
+1

**Comment by ianclegg** [comment]
+1

**Comment by yufeiminds** [comment]
+1

**Comment by DooVDe** [comment]
+1

**Comment by SystemZ** [comment]
I have this too

**Comment by vinniecent** [comment]
+1

**Comment by chadslater** [comment]
+1

**Comment by YadrovSergey** [comment]
+1

**Comment by DannyvdSluijs** [comment]
Same for me here.

**Comment by mnicolas80** [comment]
+1

**Comment by rottmann** [comment]
Fixed in 0.13.2

**Comment by DannyvdSluijs** [comment]
Great that this is solved.
